### DISCUSSION TITLE
Proposal: Fallback for the transcript tag

### DISCUSSION THREAD

**Discussion by ryan-lp** [Podcastindex-org/podcast-namespace#549] [Spec Proposal]
This is both a proposal and an open letter to podcast app developers to please consider supporting a modest fallback to the `<transcript>` tag that will not deny listeners access to transcripts, even when a hosting platform hasn't supported them yet. 

The proposal is simple. When a podcast app loads a transcript:

1. First look for a transcript link in the `<podcast:transcript>` tag.
2. If one is not found, then look for a transcript link in the description, pattern matching against any URL that ends with a `.srt` extension. (For the uninformative JSON extensions, see #452)

Lagging adoption by hosting platforms and podcaster apathy should not be reasons to deny a listeners access to transcripts in the present, and so we need a pragmatic way to balance the immediate needs of certain listeners with the desire to roll out the namespace standard universally in the long term. Although the march for universal adoption of the namespace should continue on, this proposal is about _ensuring comprehension and accessibility for all listeners in the present_. 

-----------

### Background

Over the past few years, I have been approaching podcasters to ask if they would consider publishing transcripts via this new tag, and the response has almost universally been along these lines:

- Sounds interesting!
- What's a tag, then, and where do I find it?
- My host doesn't have that.
- No I don't want to switch hosts.

"I see. What if you just add a link to the transcript in your episode description? Would be willing to publish transcripts then?"

- Yes, I could do that.

I am hearing definite interest in transcripts and a willingness from podcasters to add a link to transcripts in at least SOME way, whatever they can manage within their current setup, but as yet, there are no apps that actually support this, and so even if they did add a link in this way, the podcast apps would not yet actually take advantage of it and show those transcripts to the listener. In theory, it is a small step from here to there for any developer who wants pick up these links.

I am not going to give up the advocacy work of trying to get more hosting companies to support the namespace, however in the meantime, my job would be a lot easier if my typical dialogue (above) could finish with "Great! If you do that, here are a few podcast apps you can recommend to your listeners that will pick up and display these transcripts."

**Comment by nathangathright** [comment]
IMO, putting links to SRT files in the show notes feels hacky and very hostile to the listener. But if podcasters are willing to link to a transcript somewhere, I’d love to see transcript providers support some sort of markup on their listener-facing web pages that podcast apps could crawl. For example, if a podcaster links to a public Descript project, those webpages already include:

```
<meta property="descript:subtitles" content="[link to WebVTT file]">
<meta property="descript:transcript" content="[link to JSON file]">
``` 

I’ve built a feature for Steno.fm that can recognize those links, crawl those pages, and display those transcripts alongside the audio. The major downside to this approach is that apps have to develop heuristics for which links to crawl, but I think asking podcasters to include something like a `rel=transcript` attribute on links is too onerous.

Maybe there should be a `<meta property="podcast:transcript">` spec. Or alternatively, more awareness and adoption of existing Schema.org structured data like:
```
{
   "@context": "https://schema.org/",
   "@type": "PodcastEpisode",
   "url": "http://www.relay.fm/inquisitive/27",
   "name": "#27: Behind the App #1: History",
   "audio": {
     "@type": "AudioObject",
     "contentUrl": "http://www.relay.fm/inquisitive/Inquisitive_027.mp3",
      "caption": {
        "@type": "MediaObject",
        "encodingFormat": "application/x-subrip",
        "contentUrl": "http://www.relay.fm/inquisitive/Inquisitive_027.srt"
      }
   }
}
```

**Reply by ryan-lp** [reply]
Thanks for the suggestions and further ideas. I think links in the listener facing page are also feasible.

As a bit of a reality check, the deeper the link is, the less likely it is to be crawled and hence the less impact it will have on the listeners who need those transcripts, and the more metadata or structure we need to place around the link, the less likely it is that the podcaster will be bothered to do it.

* The average podcaster on a platform _without native support for transcripts_ is likely not going to go out of their way to support transcripts by adding complicated metadata structures manually.
* And the average podcast app is likely not going to search 3 levels deep for a transcript link.

But the transcript advocate's job will be more successful if they could just say to the podcast creator: "All you need to do for the transcript to start showing up to your listeners is to place the link anywhere in your episode description, or you could even add it anywhere in the listener facing page for the episode. And if you later choose to switch to a hosting platform that has native support for transcripts, you'll also gain the option to share that transcript link directly with the podcast apps without displaying it in your description." (Thus providing incentives to consider switching hosts without outright requiring it from the start. We have to allow a gradual transition here.)

That approach could be considered the lowest common denominator, because it is easiest for both podcasters to implement and for crawlers to implement. I'm not saying other options are invalid by any means, just that fewer apps will support it the more sophisticated this gets, and fewer podcasters stuck in this predicament will also implement it the more sophisticated this gets.

**Reply by ryan-lp** [reply]
P.S. The idea of picking up the transcript "wherever" the podcaster is able to manage putting it (if they're on a host that does not support the `<podcast:transcript>` tag) is similar to the idea of podcast feed verification where in practice we are going to have to pick up the verification token anywhere in the feed, "wherever" the podcaster is able to manage putting it (if they're on a host that does not support the `<podcast:txt>` tag). It doesn't need to be a standard, it can be considered a sort of meta standard or quasi standard, but it just reflects the reality of needing a gradual path to get from where we are today on adoption to more widespread namespace adoption in the future.

**Reply by nathangathright** [reply]
I haven’t released this feature yet, but Steno.fm will be able to serve as the host of the transcript file and the link podcasters include in their show notes. Podcasters will be able to upload their own transcripts and get a link to include in their show notes. Once they’ve verified they’re the podcaster by publishing that link in the show notes, the transcript will appear for all visitors.

![image](https://github.com/Podcastindex-org/podcast-namespace/assets/3632071/02915904-03ce-44ee-8e7f-c58c16863703)

Ideally, other podcast apps could recognize my `steno.fm/transcript` links, crawl them, and retrieve the transcript file URL. So I just need to support one of the ways to do that.

**Comment by jamescridland** [comment]
By offering a shim alternative to "my podcast host can't be bothered to let me link to my transcripts properly", are we not perpetuating the issue that some podcast hosts can't be bothered to let podcasters link to their transcripts properly?

* Sounds interesting!
* What's a tag, then, and where do I find it?
* My host doesn't have that.
* No I don't want to switch hosts.

... then send your host's support team this email, which contains:
a) the specification
b) a suggested simple UX for the specification
c) more in-depth, suggested transcript providers including rev share for the podcast host
d) the legal position of deliberately denying the ability to link to transcripts
e) a list of podcast hosts who support the transcript tag

I'm nervous about producing shims to cover up for podcast hosting company ineptitude. Surely the right thing to do is to pressure podcast hosts to implement at least the simplest version of this tag (which is to paste a URL into a box).

**Reply by ryan-lp** [reply]
@jamescridland please understand that I have been doing exactly that advocacy work in approaching individual podcasters, hosting companies, government agencies and businesses who self host their podcasts, for the past 2.5 to 3 years (since the beginning of the podcast namespace!). It is all fine and well to keep working toward that vision, but at the same time, literally YEARS have gone by where listeners of podcasts have been left behind because the market forces don't serve them. We can take the purist view where we uncompromisingly accept nothing short of complete conformance to a standard we are imposing on others, and listeners who rely on transcripts suffer for years, OR we be realistic and allow a gradual transition towards this ideal future that encourages adoption by providing options for everyone on every host.

Organisations that I have spoken to -- when they have replied (and the large ones with market share actually don't) -- have shown genuine interest in the developments surrounding the transcript tag, but since it requires engineering work, they usually either don't have the resources to treat it as a priority or they have other legal issues to contend with. For example, SBS Australia would like to do it, but for the entirety of last year they had no resources available due to the world cup demanding all of their attention, and additionally they have a legal complication where they use licensed audio content where they only have a license to it in audio form and not in textual form. Feel free to have a chat with them yourself, but don't assume that I haven't already done so myself. Maybe you'll have more sway.

I have also used the help of my friends in other countries to translate emails so that I could communicate with hosting companies in other countries, so please understand the lengths I have gone to to further the cause of the podcast namespace before thinking that the answer is as simple as doing something that I am already doing.

Sadly, this all misses the point:

The point of this plea is to immediately serve the listeners who are being left behind. Listeners who depend on transcripts are not a market force. Competition in the hosting market place serves the podcaster, not the listener. But it is not the podcaster, primarily, who is enticed by transcripts, it is the listener.

This is why when the podcast index started years ago, I realised that the only way to truly serve the listeners was to create a free podcast transcription service that made it easy for podcasters who use Anchor (Spotify) to say YES to transcripts, and hard for them to say no. These are podcasters who have no interest in transcripts themselves, and who are very happy to stay on their current hosting platform. I will release this free transcription service after the first big podcast app for Android or iOS supports a way to pick up transcript links from description fields where the podcaster is unable to provide the link in any other way.

The right to have access to transcripts is something that goes higher than the namespace itself, and it so it should not be tied down by purist views of the namespace where only 100% compliance with "our" standard is acceptable right from day 0, and other ways of trying to help listeners who are being left behind in the meantime are unacceptable in some way. It is possible to support the transcript tag entirely, but at the same time to recognise that it is not the only way in which we can help listeners by delivering transcripts into their hands.

I think we could make much faster progress on the transcript front if we allowed for a gradual transition. Imagine all of the gradual adoption we could pick up from podcasts hosted on Spotify if we did not have such a black and white view of things? If we compromised a little, and dramatically increased the number of transcripts out there which still conform to "our" standard transcript formats (because it is our initiative), we are opening a pathway to more hosts adopting the full standard in the future. For example, when a podcaster does decide to switch from Anchor/Spotify to say transistor.fm or blubrry, AND if that podcaster has already started partial adoption of transcripts by including the link in the description, the new Podcasting 2.0 host would be able to suck that up and put it in a proper transcript tag, and we would end up with far more widespread availability of transcripts than we would otherwise.

**Reply by jamescridland** [reply]
I don't disagree with anything you say; but it'll essentially lead to podcast hosting companies not implementing any new tags because we've found a way round their inability to deal with a simple episode value. I'm nervous about doing that.

Your example of SBS Australia - there are three things in there:
a) can they produce the transcripts? That's a big heavy tech lift, and they may not be able to
b) are they able to produce the transcripts for rights reasons? That's a difficult piece of legal work, and they may not be able to
c) can they support a podcast:transcript tag? They're hosted on StreamGuys, so no tech lift by SBS required

This shim fixes the third "thing". But the other two are probably more difficult.

**Reply by ryan-lp** [reply]
> a) can they produce the transcripts?

They actually do produce subtitles for broadcast television, so I'd imagine they have some capacity.

> b) are they able to produce the transcripts for rights reasons?

SBS wrote:

> We broadcast content that we may have rights to in audio format - but not textually.

You wrote:

> c) They're hosted on StreamGuys

I did get this from SBS back in November (after ongoing discussions that had lasted more than a year):

> Transcriptions is currently on our roadmap - but we're currently focussing on migrating our sites to a new CMS platform.  Once finalised we will look to deploy transcription and other functionality.

I'm not sure but I'm guessing that perhaps they have by now completed that migration and StreamGuys is their new CMS platform. I have noticed over this timeframe that some of their podcasts have suddenly gained dynamic ad insertion, which I guess was due to their use of a new platform. I also realised we now have to contend with how dynamic ad insertion is going to mess up transcript timestamps in ways I outline in #556 .

> This shim fixes the third "thing". But the other two are probably more difficult.

My reason for bringing up the SBS case was mainly with regards to the legal aspect. But with regards to (c) I am speaking mainly of hosting companies like Spotify/Anchor, and their equivalents in other countries around the world. For example, seesaa.net in Japan provides free podcast hosting, but they are impossible to get in touch with despite my efforts, and so I have been unsuccessful in even getting the transcript tag onto their agenda. I have a feeling that their software platform is now big and fragile and they wouldn't want to risk touching it.

> but it'll essentially lead to podcast hosting companies not implementing any new tags

Smaller hosting companies will continue to do whatever they can to get a competitive advantage which means offering features that Spotify lacks, so I'm not worried if we remove one reason for why Spotify should adopt the namespace. They weren't going to adopt it anyway. However, what this does do is:

1. Helps the listeners who are currently being left behind
2. Increases the number of standards-compliant transcripts out there so that when the time comes for podcasters to migrate to other platforms that do implement the namespace, they can suck them into the standard transcript tag, and there will actually be more adoption of the transcript tag in the end.

**Reply by jamescridland** [reply]
I think I'll leave it here:

> it'll essentially lead to podcast hosting companies not implementing any new tags because we've found a way round their inability to deal with a simple episode value

Listeners won't be left behind if podcast hosts adopt the tag. It's really simple for them to do so.

If we put a shim in place for every new podcast namespace feature, we kill any chance of the podcast namespace features being properly implemented by podcast hosts. So, I'm against the idea of shims for anything - and @daveajones has also pushed back on the idea too.

**Reply by ryan-lp** [reply]
This plea does not actually address those who pen the namespace, or the podcasters, or the hosting platforms. It is only natural for each of those parties to have their own interests at heart. So please feel free to continue having whatever interests you have. However, in the interest of the listeners who rely on transcripts, things look very different. The vision of pure 100% namespace conformance over anything else doesn't mean anything to the listener, what means something is having immediate access to transcripts now, not in 3 years, or 6 years or 10 years.

In almost 3 years since I have been advocating for the transcript tag, none of the podcasters I have spoken to have actually been persuaded to move off Spotify/Anchor just so they can get transcripts, and good luck trying to convince Spotify/Anchor or seesaa to add native support any time soon. That is the reality of it. And so in the interest of the listeners, none of what's in your own interests here actually represents what's in the listener's interests. That's fine, but the discussion is not over for those who still have interests, it's just that you yourself can "leave it here" if, as you say, you wish.

Fortunately this is something that app developers are in a position to do something about, and I am grateful that @nathangathright has taken on the task of picking up transcripts wherever they may appear, and not "just" when they conform to THIS particular standard. The podcast namespace is not the only way of doing things, it is not the law, people need to be able to serve the listeners in whatever way they can manage given their circumstances. The podcast namespace is not even the only standard. A podcast app does not need to be limited to what only one standard says they should be doing, a podcast app can support all of the different standards out there if it wants to. Some people use the `podcast:chapters` tag, some people embed the chapters in ID3 tags. In the podcast namespace's interest, it obviously wants everyone to use its own tag that it is prescribing for everybody, but in reality there are other standards and other ways of doing things, and apps are able to support chapters wherever they may appear, and so too should they be able to support transcripts wherever they appear.

This is a plea to the app developer in the interests of listeners, no one else.

**Reply by ryan-lp** [reply]
To paraphrase someone:

_The podcast namespace shouldn't necessarily be aiming at being the only podcast standard and instead be the best podcast standard._

**Reply by jamescridland** [reply]
Thanks, Ryan. I totally understand your point of view.

I'm still a bit fundamental about this - for me, the best outcome for the listener would be for the transcript tag to be supported correctly and consistently by podcast hosts. So, I don't believe, personally, that this is a proposal that **the podcast namespace** should be taking forward.

That doesn't mean that it isn't a good idea, nor that individual podcast apps would be wrong to implement it should they wish. Just that - personally - I don't believe that the podcast namespace is best served by shims or kludges that could potentially slow the adoption of the proper namespace tag for a transcript. And therefore, if you're trying to propose that this is adopted by the podcast namespace, I'd be uncomfortable for this to be taken forward as a proposal by the namespace group.

Nathan's idea of `meta` tags are unlikely to be available to most podcasters, who don't have access to the HTML code that their podcast provider produces: https://rss.com/podcasts/james-cridland/1033196/ isn't possible for me to add a meta tag into. It's also unlikely that the likes of Spotify for Podcasters allows `rel=""` type links either. So we're probably coming back to simple 

```
<a href="https://podnews.net/audio/podnews230714.mp3.srt">
  Transcript
</a>
```

...and that's about as good as we'll get, I guess. I'm a bit uncomfortable about that, since the resulting file is computer code that listeners shouldn't see; and I would prefer

```
<a href="https://podnews.net/audio/podnews230714.mp3.srt"></a>
```

...i.e. simply an empty link (because we don't actually want anyone clicking them, do we?). An empty link like the above is valid HTML5, with the benefit that it would not be inadvertently clickable by a listener, and should be essentially invisible to them.

I really want creator-produced transcripts to get adoption. However, I also believe that our best chance is to get the namespace tag correctly adopted by podcast hosting companies - and anything that gets in the way of that should be resisted by this group.

**Reply by ryan-lp** [reply]
You seem to have missed a few things. This is not a namespace proposal, it is an open letter to app developers to provide a fallback. It is similar to how feed owner verification will not work in practice if we only support the verification token being in the `txt` tag, because not every hosting platform supports the `txt` tag and so apps will need to search for the verification token anywhere in the feed. The fact that apps do this is not because it is part of the podcast namespace standard, but if apps start collectively doing this in order to support legacy platforms, then it becomes a sort of quasi standard. It does not need to be something that is officially part of podcast namespace spec, but rather it is a convention that apps can follow in practice to handle where the spec meets reality.

> and anything that gets in the way of that should be resisted by this group.

Don't you think there is a bit too much "us" vs "them" going on here?  Perhaps you did not recognise the irony of the paraphrased quote I gave in my last comment?

The original quote is of you suggesting on stage to Pinol that "Spotify shouldn't necessarily be aiming at being the **_only_** place to download podcasts and instead be the **_best_** place to download podcasts."

Yes, fine words. If only Spotify would not think of itself as the only podcast place in town, unilaterally making decisions on the way it thinks podcasting should be, and hence considering its own way the only way to do podcasting that's worth considering. If only it would acknowledge other ways of doing podcasting such as the podcast namespace. Right?

Yes, let's hope, * ahem *, "Spotify" drops this us-vs-them attitude * ahem *.

Meanwhile, Podcast Addict supports both `podcast:chapters` AND Spotify's way of doing chapters in the RSS feed (either embedding the chapters in the description, or putting them in the `psc:chapters` XML element). Bravo Xavier for supporting chapters wherever they may appear, and without picking a side at the exclusion of the other side.

It is OK to have your own standard, but let's not say that "to be one of us you must be anti- any other competing standard". Because if that is the case, then I am NOT one of "this group".

P.S. I just had to point out the irony, but that is still besides the point. By telling "this group" that they should "resist" any effort other than the podcast namespace to bring transcripts to the listener, it hurts the listener. Yes it's just pulling from Spotify's playbook, but it's also mainly just hurting the listener. Again to paraphrase, the podcast namespace should not aim to be the "only" standard, it should just aim to be the "best" standard. Play nice with others who are also making independent efforts for the benefit of listeners.

**Reply by jamescridland** [reply]
> This is not a namespace proposal

You're posting it in the podcast namespace discussion area, and it's marked "proposal".

🤷

**Reply by ryan-lp** [reply]
Yes, it's a proposal, but not a namespace proposal. I'm pretty sure I didn't select the type of proposal that governs the namespace itself. There is no actual namespace tag being proposed here, so I selected the other type of proposal (from the menu) that is not confined to a tag. Although as I said above, I'm also fine to treat this as a quasi standard like the `txt` fallback.

**Reply by ryan-lp** [reply]
@jamescridland let's assume for a moment that your approach to this is the way to go here. That is, even though it would be possible for apps to support transcripts on all legacy platforms in a matter of weeks via the proposed method, you say that "this group" should actually resist this idea, and effectively withhold listeners' access to transcripts for certain podcasts. You believe that ensuring the podcast namespace is the ONLY way that transcripts are done, at the exclusion of other ways (i.e. us-vs-them), is actually more important than the listener's right to have access to podcast transcripts . OK, so let's go with that assumption for the moment.

Now, pick 3 random podcasts from 3 different countries hosted on popular and free legacy platforms from those respective countries, where I am also waiting in the wings ready to donate transcripts to each of them once the apps can actually crawl them. My question for you is, how many more years (beyond the 3 years I've already been at it) before the listeners are served, if we follow your uncompromising approach? And do you truly think their wait will be worth it if it means that the podcast namespace gets to be the only standard?

Do you truly think that having more than one way to crawl transcripts will be detrimental? Was it detrimental for chapters that Podcast Addict supports your preferred approach and also Spotify's preferred approaches?

Is the mission to have your preferred way be the only way? Or is the mission to actually deliver transcripts, chapters, etc into the hands of listeners in whatever ways we can get the apps to support?

**Reply by jamescridland** [reply]
I _personally_ don't feel that this is a _podcast namespace proposal_ that this group should support, as I've made very clear. By adding a shim like this, it's my belief that this will delay getting the podcast transcript feature in the namespace fully adopted by the whole podcast ecosystem. I'm entitled to a personal view, and nothing you've said so far has changed my mind.

You're suggesting that I don't care about the listener. That's wrong (and not a little insulting): I want to be in a situation where podcast hosting companies support the ridiculously easy podcast:transcript tag. I don't want to give podcast hosting companies an excuse not to support transcripts, which is explicitly what your suggestion offers them - an easy way out that doesn't deliver the right thing for the listener, which is full support for the transcript feature.

> Was it detrimental for chapters that Podcast Addict supports your preferred approach and also Spotify's preferred approaches?

I'm a bit confused about this point - but perhaps it's worthwhile highlighting, again, that this is my personal view. I'm not speaking on behalf of the new podcast namespace - just speaking on behalf of... me.

Since you brought it up - my personal view about chapters is to use ID3 tags, which are widely supported. I don't actually think the chapters JSON file does anything better than the ID3 standard: and specifically does a number of things worse (including support for DAI). I think the new podcast namespace approach to supporting chapters is a mistake. OK?

**Reply by ryan-lp** [reply]
I'm not suggesting that you don't care about the listeners, only that given the choice between:

1. making the listeners who depend on transcripts wait 6 years, while you convince hosts in all of these different countries to implement your preferred standard, and
2. giving them an option now so they don't have to wait 6 years

you would still choose scenario 1, because you would resist scenario 2 at all costs, which sadly includes the cost of the listener having to wait 6 years. So while you care about listeners' right to transcripts, there's apparently something that you care about even more than listeners' right to transcripts, and that is to resist scenario 2 at all costs. You would not compromise on that for anything because, as you said, you are fundamental about it.

The fact is that listeners who rely on transcripts don't actually care about which approach is used. What they care about is to be able to actually comprehend the podcast with the aid of transcripts. Not in 6 years, but as soon as possible. As someone with mobility issues who struggles with stairs (although not being wheelchair bound), I also often have to deal with the problem of getting into a building, even my own apartment building. There is a way I can get in by walking a long way around the building, and through a corridor, although that is beyond the walking distance I can cope with. Now when I spoke to the building manager, did he give me some resistance and say that I should not be able to easily get access to the building until they save up enough money to do some construction work to build a ramp for this side of the building as well as the back way? Well no, because I proposed him an immediate alternative: give me permission to access the building through the car park entrance. And although it was not a "proper" solution, I was very grateful that I was then able to enter and exit my own apartment building without causing more injuries to my hips and knees. He showed that he cared more about me than about having the perfect solution, because he factored the timescale into it and how the waiting period would actually cause more harm.

Now where does this 6 years come from? Well your response didn't actually address the time frame challenge I put to you. So, I am using my own estimate: Your approach will take 6 years to deliver transcripts into the hands of the listeners of these 3 podcasts from 3 random countries. After 3 years in the English markets we are getting some adoption, although it is still a relatively small amount of adoption. But In many other countries this process of adoption has barely even begun if at all, and is therefore lagging 3 years behind.

Meanwhile, I think that all it takes for option 2 is for at least ONE app to crawl the transcript link wherever it finds it, and then those waiting listeners would have an option available to them that until now has been denied to them. I think that can be realistically be achieved in 1 month or less, not 6 years.

I will also make a prediction that due to the ease of at least one app implementing this compared to the difficulty of all hosts implementing the transcript tag that it's inevitable that both of these approaches are going to coexist until the need for an alternative evaporates. As someone who is only happy with the perfect option, you won't actually be able to prevent people from publishing transcripts in different ways from your preferred way if ultimately people need transcripts however they can manage to get them, and you won't be able to prevent apps from crawling transcripts wherever they appear, if it means we can deliver more transcripts into the hands of listeners. This is the only metric that means anything.

**Reply by jamescridland** [reply]
I can only repeat what I've said - that a shim/kludge that allows podcast hosts to avoid properly implementing something is not in the best interest of the podcast namespace, and not, ultimately, in the best interest of listeners since it slows down adoption of standards.

I am not, however, anyone other than "another bloke with a point of view". If you're able to carry others with you, whether here or elsewhere, then go for it.

I don't intend to repeat myself further.

**Reply by ryan-lp** [reply]
You've certainly made your position clear, but you've also not addressed the time frame challenge to your position. Rather than repeat the same points over, one way you could have avoided repeating yourself would be to actually comment on those challenges that you have until now side stepped. But that's OK. You're just an ordinary bloke with a point of view. It's not as though you have a platform where you have the privilege to influence future directions of the industry, such as to publicise this open letter to app developers, for example.
